### DISCUSSION TITLE
Wayland when.

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [wwmm/easyeffects#3399] [General]
Only X11? rlly? sad. Wiki says so.
https://github.com/wwmm/easyeffects/wiki/Headless-Operation

**Comment by wwmm** [comment]
EasyEffects already works on Wayland. I am using wayland right now. Wayland and X11 support is done at the graphical toolkit level and not on our side.

That being said headless operation may require something more from the toolkit. As I do not use EasyEffects in headless mode I do not know.

**Reply by thepragmaticmero** [reply]
oh ok. Yeah I tried without xwayland and it works~ The part about headless got me confused

**Reply by wwmm** [reply]
>The part about headless got me confused

I see. That guide was added by an user a few years ago. As EasyEffects is a gui app and not a true service some kind of fake graphical environment is needed for headless mode. If something different is needed for wayland or not I have no idea. Maybe the tutorial works on wayland as it is.
